ui/pages/token-allowance/token-allowance.js

```diff
@@ -101,6 +101,9 @@ export default function TokenAllowance({
   const { hostname } = new URL(origin);
   const thisOriginIsAllowedToSkipFirstPage = ALLOWED_HOSTS.includes(hostname);
 
+  const [customTokenValue, setCustomTokenValue] = useState(
+    dappProposedTokenAmount,
+  );
```

Comment:
hi @NiranjanaBinoy, I'm having trouble differentiating between `customTokenValue` and `customTokenAmount`. Can you please explain?

Reply:
Sorry for the confusion, `customTokenValue` was the state variable that holds the custom value we entered for the spending cap, and `customTokenAmount` is the value from the state. I have renamed `customTokenValue` to `customSpendingCap` for more clarity.